src/config.c

```diff
@@ -377,6 +377,38 @@ int get_conf_parameter_whole(const char *buf,long *pos,long buflen,char *dst,lon
   return i;
 }
 
+int get_conf_parameter_quoted(const char *buf,long *pos,long buflen,char *dst,long dstlen)
+{
+    int i;
+    if ((*pos) >= buflen) return 0;
+    // Skipping spaces after previous parameter
+    while ((buf[*pos] == ' ') || (buf[*pos] == '\t'))
+    {
+        (*pos)++;
+        if ((*pos) >= buflen) return 0;
+    }
+    // first quote
+    if (buf[*pos] != '"')
+        return 0;
+    (*pos)++;
+    // TODO: escape quotes (\", \\)
```

Comment:
Still a todo left in

Reply:
I dont want to do it now. It should be clarified are we expecting `".tga`

Reply:
TGA wiped out

Reply:
fixed